### PR TITLE
Feature/321 smooth animations

### PR DESCRIPTION
**Pull request by juliaslaats**
## Description

Added some smooth animations to make DeX look more alive

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [x] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Please look at the animations I've added and check that they work well. This is what I've added: 

**On homepage**
- Spinning animation search button
- Search bar grows when typing in it
- Scaling on hightlight cards
- Lift-up of 'Give Feedback' button

**On overview**
- Spinning animation search button
- Growing of the project-cards
- Loading animation (both for the list and the individual project)

**On login**
- Scaling of login buttons

**Extra thing I did**
I noticed that the phone on the svg on the login page was still in a colour that wasn't visible on the background, so I changed that to a darker color.

I also added this wiki page that other people can use when making new pages: https://github.com/DigitalExcellence/dex-frontend/wiki/How-to-use:-Animations

## Link to issue

Closes: #321

### CODE REVIEW

**Review by liekevdvoort — APPROVED** [review]

**Review by liekevdvoort — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added update highlight functionality - [#294](https://github.com/DigitalExcellence/dex-frontend/issues/294)
```
Comment (Brend-Smits) [inline CHANGELOG.md:13]:
```suggestion

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+- Added animations on clickable items and while project pages are loading [#321](https://github.com/DigitalExcellence/dex-frontend/issues/321)
```
Comment (Brend-Smits) [inline CHANGELOG.md:14]:
```suggestion
- Added animations on clickable items and while project pages are loading - [#321](https://github.com/DigitalExcellence/dex-frontend/issues/321)
```

**Review by Brend-Smits — APPROVED** [review]
Looks great to me! Small changes in the changelog and you're good to go!

**Review by Brend-Smits — APPROVED** [review]

**Review by DaveyHurkmans — APPROVED** [review]
Looks Great nothing out og the ordinary :) Well done!

**Pull request merged**